frontend/src/components/regions/CardDetails/Review.js

```diff
@@ -68,26 +80,43 @@ const Review = ({ review }) => {
         className={classes.cardHeader}
       />
       <CardContent className={classes.cardContent}>
-        {/* <Typography className={classes.cardFont}>
-          <ShowMoreText
-            lines={1}
-            more={<ExpandMore />}
-            less={<ExpandLess />}
-            onClick={onClick}
-            expanded={expand}
-            width={0}
-            truncatedEndingComponent={"..."}
-            > */}
-        <Markdown children={review.comments} />
-        {/* </ShowMoreText> */}
-        {/* </Typography> */}
+        {/* 
+            217 is the approximate max number of chracters that can be visible before the need for
+            a scroll. As long as a review is longer than that, show "...read more"
+        */}
+        {review.comments.length > 220 ? (
+          <div>
+            <Button
+              type="button"
+              size="small"
+              className={classes.iconAlignment}
+              onClick={() => setOpenReviewPopUp(true)}
+            >
+              <Typography variant="caption" className={classes.readMoreStyle}>
+                ... Read more
+              </Typography>
+            </Button>
+            <Typography noWrap>
+              {cleanMarkdown(review.comments.trim())}
+            </Typography>
+          </div>
+        ) : (
+          <Typography>
+            {review.comments}
```

Comment:
Why are you calling cleanMarkdown sometimes and not other times?

Reply:
Resolved